frontend/javascripts/oxalis/model/bucket_data_handling/bounding_box.ts

```diff
@@ -47,9 +47,15 @@ class BoundingBox {
   });
 
   containsBucket([x, y, z, zoomStep]: Vector4, resolutionInfo: ResolutionInfo): boolean {
-    const { min, max } = this.getBoxForZoomStep(
-      resolutionInfo.getResolutionByIndexOrThrow(zoomStep),
-    );
+    /* Checks whether a bucket is contained in the active bounding box.
+     * If the passed resolutionInfo does not contain the passed zoomStep, this method
+     * returns false.
+     */
```

Comment:
This method is only used by `DataCube.isWithinBounds` which itself also checks for `this.cubes[zoomStep] == null`. So, the new behavior here is in line with the existing behavior on the call site.
The other usage of `containsBucket` was removed due to the `NullBucket` consolidation.